client/src/api/api.tsx

```diff
@@ -151,3 +151,51 @@ export const executeInvocation = (invocation: InvocationType) => {
     body,
   });
 };
+
+//just for development
+export interface ProjectType {
+  name: string;
+  id: string;
+  isOpen: boolean;
+}
+
+//will get all the user projects from the backend
+export const getProjects = (): ProjectType[] => {
+  const projectsArray: ProjectType[] = [
+    { name: "project1", id: "1", isOpen: false },
+    { name: "project2", id: "2", isOpen: false },
+    { name: "project3", id: "3", isOpen: true },
+    { name: "project4", id: "4", isOpen: false },
+  ];
+  return projectsArray; //just for development checking
+};
+
+export const GetChangeBetweenProjects = (projectClicked: any) => {
+  // Change between projects logic here
+  return console.log("move to project", projectClicked.target.value);
+};
+
+//Delete project
+export const DeleteProject = (e: React.MouseEvent, projectName: string) => {
```

Comment:
Functions should start with a lower case letter. This is Relevant to all the functions you have created.